database/seeds/SampleDataSeeder.php

```diff
@@ -32,7 +32,45 @@ protected function channels()
     {
         Channel::truncate();
 
-        factory(Channel::class, 10)->create();
+        collect([
+            [
+                'name' => 'PHP',
+                'description' => 'A channel for general PHP questions. Use this channel if you can\'t find a more specific channel for your question.',
+                'archived' => false,
+                'color' => '#008000'
+            ],
+            [
+                'name' => 'VueJS',
+                'description' => 'A channel for general VueJS questions. Use this channel if you can\'t find a more specific channel for your question.',
+                'archived' => false,
+                'color' => '#cccccc'
+            ],
+            [
+                'name' => 'Laravel Mix',
+                'description' => 'This channel is for all Laravel Mix related questions.',
+                'archived' => false,
+                'color' => '#43DDF5'
+            ],
+            [
+                'name' => 'Eloquent',
+                'description' => 'This channel is for all Laravel Eloquent related questions.',
+                'archived' => false,
+                'color' => '#a01212'
+            ],
+            [
+                'name' => 'VueEx',
+                'description' => 'This channel is for all VueEx specific questions.',
+                'archived' => false,
+                'color' => '#ff8822'
+            ],
+        ])->each(function ($channel) {
```

Comment:
each() is deprecated since 7.2

http://php.net/manual/en/function.each.php

Reply:
This each method is from laravel's Collection.